tool/speed.cc

```diff
@@ -169,8 +169,8 @@ struct free_functor {
   }
 };
 
-#if defined(OPENSSL_WINDOWS)
-#define AllocAligned malloc
+#if defined(OPENSSL_WINDOWS) || defined(OPENSSL_ANDROID)
```

Comment:
Looks like you commented out the `OPENSSL_ANDROID` definition in the cmake file. Is this still needed?